src/xSTIR/pSTIR/STIR.py

```diff
@@ -806,21 +814,28 @@ def show(self, sino = None, title = None):
                 ns = len(sino)
             except:
                 raise error('wrong sinograms list')
+        
         if title is None:
             title = 'Selected sinograms'
-        if ns >= 16:
-            tiles = (4, 4)
-        else:
-            tiles = None
+        
+        #if ns >= 16:
+            #tiles = (4, 4)
+        #else:
+            #tiles = None
+        
```

Comment:
I'm not sure why you did this, but I'd rather not do a surprising change in this PR